src/queries/enum_repr_int_changed.ron

```diff
@@ -43,9 +49,14 @@ SemverQuery(
                         # - but is not the same repr(i*) or repr(u*) that we had before.
                         # This is the breaking change.
                         attribute @fold @transform(op: "count") @filter(op: ">", value: ["$zero"]) {
-                            value @filter(op: "regex", value: ["$repr_regex"])
-                                    @filter(op: "!=", value: ["%attr_value"])
-                                    @output(name: "new_attr")
+                            as_string @output(name: "new_attr")
+                            content {
+                                base @filter(op: "=", value: ["$repr"])
+                                argument {
+                                    base @filter(op: "regex", value: ["$repr_regex"])
+                                            @filter(op: "!=", value: ["%old_attr_value"])
```

Comment:
```suggestion
                                    base @filter(op: "regex", value: ["$repr_regex"])
                                         @filter(op: "!=", value: ["%old_attr_value"])
```
Another alignment fix. It's possible your editor is doing something funny with tabs and spaces, might be worth a look.

Another thing I frequently do and highly recommend: using the GitHub PR review interface to self-review your PR after opening it. The "change of scenery" in how you read the code makes many kinds of bugs pop out in a way that reading the same code in your editor doesn't.

Reply:
That's just me not paying attention to the alignment 😅 I will try doing the self-review thing next time to avoid these small mistakes.